openslides_backend/action/actions/projection/create.py

```diff
@@ -0,0 +1,17 @@
+from ....models.models import Projection
+from ...generics.create import CreateAction
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+
+
+@register_action("projection.create", internal=True)
+class ProjectionCreate(CreateAction):
```

Comment:
this action is different to https://github.com/OpenSlides/openslides-backend/pull/552

Maybe you should do it the same way to avoid conflicts.

Reply:
Resolved the conflict.